lib/hammer_cli_csv/csv.rb

```diff
@@ -3,9 +3,59 @@
+      def add_list(heading, items)
+        items.delete_if do |item|
+          if item.class == Clamp::Subcommand::Definition
+            subcommand_class = item.subcommand_class
+            subcommand_class == HammerCLICsv::CsvCommand::ActivationKeysCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::ArchitecturesCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::ComputeProfilesCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::ComputeResourcesCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::ContainersCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::ContentHostsCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::ContentViewsCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::ContentViewFiltersCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::DomainsCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::HostCollectionsCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::HostsCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::HostGroupsCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::InstallationMediasCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::JobTemplatesCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::LifecycleEnvironmentsCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::LocationsCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::OperatingSystemsCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::OrganizationsCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::PartitionTablesCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::ProductsCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::ProvisioningTemplatesCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::PuppetEnvironmentsCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::PuppetFactsCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::PuppetReportsCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::ReportsCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::RolesCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::SmartProxiesCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::SpliceCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::SubnetsCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::SubscriptionsCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::SyncPlansCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::UsersCommand
+          else
+            false
+          end
```

Comment:
That's true at the moment but the goal is to get the blacklisting shorter. :) Consider this f'ugly list my incentive.